hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -1845,7 +1845,7 @@ public void testKeysRenameFail() throws Exception {
 
   @Test
   public void testListVolume() throws IOException {
-    String volBase = "vol-" + RandomStringUtils.randomNumeric(3);
+    String volBase = "vol-list-";
```

Comment:
Can you update the comment below as well
``
 //Create 10 volume vol-<random>-a-0-<random> to vol-<random>-a-9-<random>
``
to denote the change here